doc/samtools-sort.1

```diff
@@ -75,16 +75,15 @@ instead if you need name collated data without a full lexicographical sort.
 Note that if the sorted output file is to be indexed with
 .BR "samtools index" ,
 the default coordinate sort must be used.
-Thus the \fB-n\fR, \fB-N\fR, \fB-t\fR and \fB-M\fR options are
-incompatible with
+Thus the \fB-n\fR, \fB-N\fR and \fB-t\fR options are incompatible with
 .BR "samtools index" .
 
-When sorting by minimisier (\fB-M\fR), the sort order is defined by
-the whole-read minimiser value and the offset into the read that this
-minimiser was observed.  This produces small clusters (contig-like,
-but unaligned) and helps to improve compression with LZ algorithms.
-This can be improved by supplying a known reference to build a
-minimiser index (\fB-I\fR and \fB-w\fR options).
+When sorting by minimisier (\fB-M\fR), the sort order for unplaced
```

Comment:
I don't minimisier is a word.

Reply:
Indeed not.  Good spot.  That was in the original too.  Now corrected